rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -6913,6 +6914,113 @@ void getEstimatedFeesForNextPegOutEvent_zero_pegouts() throws IOException {
         assertEquals(Coin.ZERO, bridgeSupport.getEstimatedFeesForNextPegOutEvent());
     }
 
+    private static Stream<Arguments> fedProvider() {
+        BridgeRegTestConstants bridgeConstantsRegtest = BridgeRegTestConstants.getInstance();
+        List<FederationMember> members = FederationMember.getFederationMembersFromKeys(
+            PegTestUtils.createRandomBtcECKeys(7)
+        );
+
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP271)).thenReturn(true);
+        when(activations.isActive(ConsensusRule.RSKIP385)).thenReturn(true);
+
+        return Stream.of(
+            Arguments.of(activations, bridgeConstantsRegtest.getGenesisFederation()),
+            Arguments.of(activations, new P2shErpFederation(
+                members,
+                Instant.now(),
+                1L,
+                bridgeConstantsRegtest.getBtcParams(),
+                bridgeConstantsRegtest.getErpFedPubKeysList(),
+                bridgeConstantsRegtest.getErpFedActivationDelay(),
+                activations
+            ))
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("fedProvider")
+    void getEstimatedFeesForNextPegOutEvent_zero_pegouts_after_RSKIP385(ActivationConfig.ForBlock activations, Federation federation) throws IOException {
+        // Arrange
+        BridgeStorageProvider provider = mock(BridgeStorageProvider.class);
+
+        int pegoutRequestsCount = 0;
+        when(provider.getReleaseRequestQueueSize()).thenReturn(pegoutRequestsCount);
+        when(provider.getNewFederation()).thenReturn(federation);
+
+        Coin feePerKB = Coin.MILLICOIN;
+        when(provider.getFeePerKb()).thenReturn(feePerKB);
+
+        // Act
+        BridgeSupport bridgeSupport = bridgeSupportBuilder
+            .withProvider(provider)
+            .withActivations(activations)
+            .build();
+
+        // Assert
+        int outputs = pegoutRequestsCount + 2; // N + 2 outputs
+        int pegoutTxSize = BridgeUtils.calculatePegoutTxSize(activations, federation, 2, outputs);
+
+        Coin expected = feePerKB.multiply(pegoutTxSize).divide(1000);
```

Comment:
It's probably easier to simply have the expected value hardcoded than to repeat the entire logic

Reply:
I moved this to a method so now it's only in one place. What do you think now?